src/auth/auth.cpp

```diff
@@ -85,17 +86,15 @@ std::optional<User> Auth::Authenticate(const std::string &username, const std::s
       if (FLAGS_auth_module_create_missing_user) {
         user = AddUser(username, password);
         if (!user) {
-          spdlog::warn(
-              "Couldn't authenticate user '{}' using the auth module because "
-              "the user already exists as a role!",
-              username);
+          spdlog::warn(utils::MessageWithLink(
+              "Couldn't authenticate user '{}' using the auth module because the user already exists as a role.",
+              username, "memgr.ph/auth"));
```

Comment:
```suggestion
              username, "https://memgr.ph/auth"));
```
And every other link to make it clickable in the terminal